disperser/batcher/encoded_blob_store.go

```diff
@@ -35,7 +35,7 @@ type EncodingResult struct {
 	ReferenceBlockNumber uint
 	BlobQuorumInfo       *core.BlobQuorumInfo
 	Commitment           *encoding.BlobCommitments
-	Chunks               []*encoding.Frame
+	Chunks               core.Bundle
```

Comment:
In the comment above Bundle struct,  it is intended as the collection of chunks associated with a single blob, for a single operator and a single quorum. EncodingResult contains multiple operators, we can pack frames into bundles later

Reply:
Hmm I see your concern. There is really no concept of operator at this point, but wanted to use `Bundle` so that we can reuse existing `Size` method. I'll just leave it as a slice here and convert it when it actually uses `Size` method